kolibri/content/migrations/0001_initial.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.9.1 on 2016-06-27 23:19
+# Generated by Django 1.9.1 on 2016-06-29 21:25
```

Comment:
Other than this, looks good to merge. Please revert this migration to the original and then add a new migration on top of the old one -- we need to get into that workflow or we're going to have headaches.